### ISSUE TITLE
2019-8-11

### ISSUE THREAD

**Issue by ssfc** [ssfc/wangdao#78]
291.analyze model; data is all dinosaur’s name; what is {} in python; dictionary; char to x and x to char is dictionary; change the number of iterations; examples are shuffle of data; deal with vocab_size so that it can deal with different cases; let us set vecab size= 26; strange; change 26 to temp_size; (2019-7-6)
292.separate 27 out; delete the default expression; delete num_iteration as default; done; what is n_a; number of hidden unit; change n_a without default; n_a -- number of units of the RNN cell; done; delete dino_name as default; done; delete vocab_size as default; done; (2019-7-6)
293.The reason why we deal with dinosuar name is to get familiar with RNN model and see what we can do with similarity; delete learning_rate in default; done; it says we are out of bound; done; (2019-7-6)
294.Figure out meaning of X and Y below optimize; it is an example; add vocab size to optimize; should we take note when reading papers? Yes; (2019-7-7)
295.Can we find a space to show lexical result so that we can easily debug; change source code to preprocess result; can we come up with a way to easily debug translate_source? How php echo an array? Let us store source code in a txt file; create source code txt, done; change source code to preprocess result; give up; problem is solved; (2019-7-7)
296.Syntax analysis used result of preprocessor, rather than lexical analysis; combine == and != and < and ~; give up; it seems the symbol we send to html are IDList, how they become symbols? It is done through translate symbol; can we delete translate symbol and use php to implement it? (2019-7-8)
297.Delete translate symbol in insert_row; done; delete translate symbol in delete row; done; delete translate symbol in add element; done; the reason why we need translate is that we need translate symbol is to show import library; I think we do not need combine row after translation; give up delete translate symbol; (2019-7-8)
298.Pay attention to how we deal with reversed word in syntax analyze; reserved words contain word symbol made up with letters and phrase titles; (2019-7-8)
299.Separate syntax as generate symbols and generate IDList; done; change symbol list to message; hold; (2019-7-8)
